DisguiseUnityRenderStream/Runtime/DisguiseFramerateManager.cs

```diff
@@ -0,0 +1,46 @@
+using UnityEngine;
+
+namespace Disguise.RenderStream
+{
+    public static class DisguiseFramerateManager
+    {
+#if ENABLE_CLUSTER_DISPLAY
+        public const bool Enabled = false;
+#else
+        public const bool Enabled = true;
+#endif
+        
+        static int k_FrameRateUnlimited = -1;
+        
+        public static bool FrameRateIsLimited => Application.targetFrameRate >= 0;
+
+        public static bool VSyncIsEnabled => QualitySettings.vSyncCount > 0;
+
+        public static void Initialize()
+        {
+#if !ENABLE_CLUSTER_DISPLAY
+            RemoveFrameLimit();
+#endif
+        }
+
+        public static void Update()
+        {
+#if !ENABLE_CLUSTER_DISPLAY
+            if (VSyncIsEnabled)
```

Comment:
Not sure I like this. You're going to flood the logs. Maybe we should do it every X frames instead?

Reply:
Now checking every frame but warning only once per session